pkg/render/fluentd.go

```diff
@@ -427,6 +424,43 @@ func (c *fluentdComponent) envvars() []corev1.EnvVar {
 					},
 				)
 			}
+
+			// When LogTypes is not defined explicitly, the default is to enable
+			// all the log types.
+			if syslog.LogTypes == nil {
+				envs = append(envs,
+					corev1.EnvVar{Name: "SYSLOG_AUDIT_EE_LOG", Value: "true"},
+					corev1.EnvVar{Name: "SYSLOG_AUDIT_KUBE_LOG", Value: "true"},
+					corev1.EnvVar{Name: "SYSLOG_DNS_LOG", Value: "true"},
+					corev1.EnvVar{Name: "SYSLOG_FLOW_LOG", Value: "true"},
+					corev1.EnvVar{Name: "SYSLOG_IDS_EVENT_LOG", Value: "true"},
+				)
+			} else {
+				for _, t := range syslog.LogTypes {
+					switch t {
+					case operatorv1.SyslogLogAuditEE:
+						envs = append(envs,
+							corev1.EnvVar{Name: "SYSLOG_AUDIT_EE_LOG", Value: "true"},
```

Comment:
Could we extract these values as constants ? Seems like they are being used in multiple places.

Reply:
It's only defined in two places now (here and then in the test). 

I'm going to leave it alone because it looks like the convention in most of these is to use the string value, so I want to be consistent with what exists.